bin/reth/src/commands/node/events.rs

```diff
@@ -357,13 +385,19 @@ where
                     );
                 }
             } else if let Some(latest_block) = this.state.latest_block {
-                info!(
-                    target: "reth::cli",
-                    connected_peers = this.state.num_connected_peers(),
-                    %freelist,
-                    %latest_block,
-                    "Status"
-                );
+                let now =
+                    SystemTime::now().duration_since(UNIX_EPOCH).unwrap_or_default().as_secs();
+                if now - this.state.latest_block_time.unwrap_or(0) > 60 {
+                    // Once we start receiving consensus nodes, don't emit status unless stalled for
+                    // 1 minute
```

Comment:
does this effectively disable this info log in normal operations?

do we want this and why?

Reply:
We wind up saying the same thing 3 different ways in the log.  once when imported, once when canoncial chain is updated, and once every time the status tick fires.  The info is already communicated via the other two means, including peer count.  The goal of the bug is to increase density of the logs and this is duplicate data.

Adding the time check brings the status back if blocks are behind.

Reply:
sounds reasonable, ty